apps/concept-catalog/pages/api/auth/[...nextauth].tsx

```diff
@@ -17,43 +17,54 @@ export const authOptions = {
       clientSecret: process.env.KEYCLOAK_SECRET ?? '',
       issuer: process.env.KEYCLOAK_ISSUER,
       idToken: true,
-      profile(profile, tokens) {
-        return {
-          id: profile.sub,
-          name: profile.name,
-          email: profile.email ?? null,
-          image: profile.email ?? null,
-          // Append the id token to the profile
-          idToken: tokens.id_token,
-        };
-      },
+      authorization: `${process.env.KEYCLOAK_ISSUER}/protocol/openid-connect/auth`,
```

Comment:
https://github.com/nextauthjs/next-auth/blob/main/packages/next-auth/src/providers/keycloak.ts#L55

Hvis jeg leser det korrekt så trenger du ikke disse så lenge du har satt issuer korrekt, den vil klare å finne frem til .wellknown ut fra issuer.

Men det ser også ut til at issuer ligger under `options`

-> 
```suggestion
      options: { issuer: process.env.KEYCLOAK_ISSUER, },
```

Reply:
Oja, OK. skal teste forslaget nå.

Reply:
eller...

eksemplelet her viser det slik du originalt hadde det: https://next-auth.js.org/providers/keycloak#example :shrug:

Reply:
Ja, riktig. Skal jeg beholde det som er?